sda-commons-server-weld-testing/build.gradle

```diff
@@ -1,3 +1,11 @@
+// we need the 'beans.xml' in the classes directory
+task copyMetaInf(type: Copy) {
+  from "${projectDir}/src/test/resources/META-INF"
+  into "${buildDir}/classes/java/test/META-INF"
+}
+
+processTestResources.dependsOn copyMetaInf
```

Comment:
Seems like this is a long existing issue. See [this](https://discuss.gradle.org/t/application-plugin-run-task-should-first-consolidate-classes-and-resources-folder-or-depend-on-installapp-or-stuff-like-weld-se-wont-work/1248) discussion from 2014.